### PR TITLE
Add RenderableShapeBuilder

### PR DESCRIPTION
**Pull request by js78**
RenderableShapeBuilder allows to visualize several renderable's properties.
Currently, it can build normal, tangent and binormal.

### CODE REVIEW

File: gdx/src/com/badlogic/gdx/graphics/g3d/utils/RenderableShapeBuilder.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+			renderable.shader = null;
+			return renderable;
+		}
+	}
+
+	private static short[] indices = new short[1];
+	private static float[] vertices;
+	private final static RenderablePool renderablesPool = new RenderablePool();
+	private final static Array<Renderable> renderables = new Array<Renderable>();
+	private static final int FLOAT = 4;
```
Comment (intrigus) [inline gdx/src/com/badlogic/gdx/graphics/g3d/utils/RenderableShapeBuilder.java]: Why is this constant called `FLOAT`?
Can you find a better name for it?

Comment (tavomaciel) [inline gdx/src/com/badlogic/gdx/graphics/g3d/utils/RenderableShapeBuilder.java]: Is it even safe assuming a FLOAT is 4 bytes? Why not query OpenGL or the System, depending on the situation?

**Comment by js78** [comment]
It's a Java side 'float' and in several places in libgdx, a constant is used.

**Comment by xoppa** [comment]
This change copies over the vertices and indices from native to managed memory one at a time, it would be more performant to copy them over all at once or to move the code to jni.

**Comment by js78** [comment]
Yes you are write, I could download all indices and vertices before looping through renderables. ?

**Pull request merged**